tests/e2e/kubetest2-kops/deployer/dumplogs.go

```diff
@@ -46,6 +46,10 @@ func (d *deployer) DumpClusterLogs() error {
 		return err
 	}
 
+	if err := d.dumpClusterInfo(); err != nil {
+		return err
```

Comment:
A thought  ... we may want to do these as best-effort (e.g. trying each dump step and returning a list of errors at the end)